test/Microsoft.AspNet.Razor.Test/TestFiles/CodeGenerator/Source/TagHelpersWithWeirdlySpacedAttributes.cshtml

```diff
@@ -0,0 +1,9 @@
+﻿@addTagHelper "something, nice"
+
+<p
+class 
+   =
+"Hello World" age   =1337
+   data-content= "@true">Body of Tag</p>
+
+<input type 	= 'text' data-content=	 "hello" />
```

Comment:
Suggest adding additional `type` and `age` variations. Not about all the scenarios with bound and unbound attributes. But good to cover more than just one example of each.

Separately use single quotes for the second `data-content` attribute to mix things up a bit more.